File: tests/fixtures/go/call_graph.go
package callgraph

// helper has fo=0 (no calls)
func helper() int {
	return 1
}

// middle has fo=1 (calls helper twice, deduplicated to 1 unique callee)
func middle() int {
	return helper() + helper()
}

// top has fo=2 (calls helper + middle)
func top() int {
	return helper() + middle()
}

File: tests/fixtures/call-graph.ts
// Fixture: known call relationships for call graph golden tests
// helper: fo=0, no calls
// middle: fo=1, calls helper (deduplication: helper() called twice = 1 unique callee)
// top:    fo=2, calls helper + middle

function helper(): number {
  return 1;
}

function middle(): number {
  return helper() + helper();
}

function top(): number {
  return helper() + middle();
}
